lib/Runtime/Base/ScriptContext.cpp

```diff
@@ -3803,11 +3834,12 @@ namespace Js
 
             scriptContext->GetThreadContext()->SetIsProfilingUserCode(isUserCode); // Update IsProfilingUserCode state
         }
+#endif // ENABLE_SCRIPT_PROFILING
 
         Var aReturn = NULL;
         JavascriptMethod origEntryPoint = function->GetFunctionInfo()->GetOriginalEntryPoint();
 
-        __try
+        __TRY_FINALLY_BEGIN // SEH is not guaranteed, see the implementation
```

Comment:
@agarwal-sandeep thanks for the test case, added under the DebuggerCommon. Testing on CI now, lets see how Windows testing is going to react to my baseline file line-endings